src/content/language/loops.md

```diff
@@ -142,6 +142,235 @@ candidates
     .forEach((c) => c.interview());
 ```
 
+### Labels
+
+A label is an identifier followed by a colon (`labelName:`)
+that you can place before a statement to create a
+_labeled statement_. Loops and switch cases are often used as
+labeled statements. A labeled statement can be referenced later
+in a `break` or `continue` statement as follows:
+
+*  `break labelName;`: Terminates the execution of the labeled statement.
+   This is useful for breaking out of a specific outer loop when you're
+   within a nested loop.
+
+*  `continue labelName;`: Skips the rest of the current iteration of the
+   labeled statement loop and continues with the next iteration.
+
+Labels are used to manage control flow. They are often used with
+loops and switch cases and allow you to specify which statement to
+break out of or continue, rather than affecting the innermost
+loop by default.
+
+#### Labels in `for` loop using `break`:
+
+The following code demonstrates the usage of a label called `outerLoop`
+in a  `for` loop with a `break` statement:
+
+<?code-excerpt "language/lib/control_flow/loops.dart (label-for-loop-break)"?>
+```dart
+outerLoop:
+for (var i = 1; i <= 3; i++) {
+  for (var j = 1; j <= 3; j++) {
+    print('i = $i, j = $j');
+    if (i == 2 && j == 2) {
+      break outerLoop;
+    }
+  }
+}
+print('outerLoop exited');
+```
+
+In the previous example, when `i == 2` and `j == 2`, the `break outerLoop;`
+statement stops both inner and outer loops. So, the expected output is:
+
+```dart
```

Comment:
Since this is terminal output from printing, consider changing the code block's language so it's clear it's not Dart code:

````suggestion
```terminal
````